canonical/juju-dashboard
# WD-9420 - Remove some references to jaas.ai

## Done

- Remove unnecessary jaas.ai references.

## QA

- Connect to a local controller.
- Click the header logo and it should take you to juju.is.
- Connect to a JAAS controller.
- Click the header logo and it should take you back to the model list.

## Details

https://warthogs.atlassian.net/browse/WD-9420
